server/src/step/step.controller.js

```diff
@@ -0,0 +1,13 @@
+import { Step } from '@/shared/database/index';
+
+const newStep = async (req, res, next) => {
+  try {
+    const step = await Step.create(req.body);
+
+    res.json({ stepId: step.id });
+  } catch (err) {
+    next(new Error());
+  }
+};
```

Comment:
we are missing adding stepIngredients, checking for units, etc. It can stay like this for now but in the future this will need to change 100%